netmap/attributes.go

```diff
@@ -0,0 +1,208 @@
+package netmap
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/nspcc-dev/neofs-api-go/v2/refs"
+)
+
+// prefix of keys to subnet attributes.
+const attrSubnetPrefix = "__NEOFS__SUBNET_"
```

Comment:
Should we export this one? It is used in `neofs-sdk-go` for placement policy selection.

Reply:
We need to check that the node belongs to a specific subnet, right?

Reply:
If placement function dives into node attributes to process subnet, then it's a version-dependent way. Actually, `NodeInfo` type should provide the functionality to work with its subnets, and placement function should use it.